packages/babel-types/src/clone/cloneNode.js

```diff
@@ -0,0 +1,35 @@
+import { NODE_FIELDS } from "../definitions";
+
+const has = Function.call.bind(Object.prototype.hasOwnProperty);
+const isNode = obj => obj && has(NODE_FIELDS, obj.type);
+
+export default function cloneNode<T: Object>(node: T): T {
+  if (!node) return node;
```

Comment:
If `T` extends `Object` it can't be falsy, unless it's `document.all` :trollface: 
  
(I guess this is defensive programming)

Reply:
Oh lol, I copied lines 6-7 from `clone`. I will leave the check here, because plug-in authors might not use Flow.